packages/components/tests/dummy/app/templates/components/alert.hbs

```diff
@@ -0,0 +1,844 @@
+{{page-title "Alert component"}}
+
+<h2 class="dummy-h2">Alerts</h2>
+
+<section>
+  <h3 class="dummy-h3">Overview</h3>
```

Comment:
Please remember to add the `id` and section link for all h3's.

Reply:
done.